insights/parsers/ip.py

```diff
@@ -13,12 +13,14 @@
 IpNeighParser - command ``ip neigh show nud all``
 -------------------------------------------------
 
+IpLinkInfo - command ``ip -s link``
+-----------------------------------
 """
 
 from collections import defaultdict, deque
 from .. import Parser, parser
 from ..contrib import ipaddress
-from insights.specs import ip_addr
+# from insights.specs import ip_addr
```

Comment:
Should not comment it either.

You should leave it here and add another one for the new added `ip-s_link`

~~~
from insights.specs import ip_addr	
from insights.specs import ip_s_link
~~~